nodejs16.x/cookiecutter-aws-sam-step-functions-sample-app/{{cookiecutter.project_name}}/makefile

```diff
@@ -0,0 +1,69 @@
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
Do we also need a `makefile` in the nodejs14.x project?

Reply:
Yes. Good catch. I have added the makefile to nodejs 14x project as well.